ramls/invoice.raml

```diff
@@ -74,7 +74,7 @@ resourceTypes:
         exampleItem: !include acq-models/mod-invoice-storage/examples/invoice_line.sample
     is: [validate]
     get:
-      is: [ pageable, validate ]
+      is: [ pageable, searchable: {description: "using CQL (indexes for invoice lines)", example: "status==\"Open\""}, validate ]
```

Comment:
According to [invoice_line.json](https://github.com/folio-org/acq-models/blob/master/mod-invoice-storage/schemas/invoice_line.json) the status is named as `invoiceLineStatus`

Reply:
description updated